From 3200dd761894f04f559c3a60b67178a9d40bf34c Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Thu, 10 Oct 2024 13:58:38 +0800
Subject: [PATCH] tests: Remove blocking_adaptation.cpp

The test failed to build with clang as below.
../../../asio-1.30.2/src/tests/../../include/asio/execution/blocking_adaptation.hpp:216:13: error: call to 'query' is ambiguous
  216 |             conditional_t<true, T, P>::query(static_cast<P&&>(p))
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../../../asio-1.30.2/src/tests/../../include/asio/execution/blocking_adaptation.hpp:213:29: note: in instantiation of exception specification for 'query<asio::execution::detail::blocking_adaptation::allowed_t<>>' requested here
  213 |       static constexpr auto query(P&& p)
      |

We can add the case back once the above build failure resolved then.

Upstream-Status: Pending

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 src/tests/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/tests/Makefile.am b/src/tests/Makefile.am
index 7161698..25f3b08 100644
--- a/src/tests/Makefile.am
+++ b/src/tests/Makefile.am
@@ -51,7 +51,6 @@ check_PROGRAMS = \
 	unit/error \
 	unit/execution/any_executor \
 	unit/execution/blocking \
-	unit/execution/blocking_adaptation \
 	unit/execution/bulk_execute \
 	unit/execution/bulk_guarantee \
 	unit/execution/connect \
@@ -244,7 +243,6 @@ TESTS = \
 	unit/error \
 	unit/execution/any_executor \
 	unit/execution/blocking \
-	unit/execution/blocking_adaptation \
 	unit/execution/bulk_execute \
 	unit/execution/bulk_guarantee \
 	unit/execution/connect \
-- 
2.25.1

